packages/flutter_tools/bin/fuchsia_builder.dart

```diff
@@ -36,6 +37,7 @@ Future<Null> main(List<String> args) async {
   executableContext.setVariable(Logger, new StdoutLogger());
   executableContext.runInZone(() {
     // Initialize the context with some defaults.
+    context.putIfAbsent(ProcessManager, () => new ProcessManager());
```

Comment:
The lists aren't the same (Fuchsia is a subset of the main executable).  Added note.